tutorials/source-modeling/plot_beamformer_lcmv.py

```diff
@@ -0,0 +1,169 @@
+"""
+Source reconstruction using an LCMV beamformer
+==============================================
+
+This tutorial gives an overview of the beamformer method
+and shows how to use an LCMV beamformer to reconstruct source
+activity.
+
+.. contents:: Page contents
+   :local:
+   :depth: 2
+
+A beamformer is a spatial filter that reconstructs source activity by scanning
+through a grid of pre-defined source points and estimating activity at each of
+those source points independently.
+The beamforming method applied in this tutorial is the linearly constrained
+minimum variance (LCMV) beamformer [1]_ which operates on time series.
+Frequency-resolved data can be reconstructed with the dynamic imaging of
+coherent sources (DICS) beamforming method [2]_.
+"""
+# Author: Britta Westner <britta.wstnr@gmail.com>
+#
+# License: BSD (3-clause)
+
+# sphinx_gallery_thumbnail_number = 5
+
+import mne
+from mne.datasets import sample
+from mne.beamformer import make_lcmv, apply_lcmv
+
+###############################################################################
+# Data processing
+# ---------------
+# We will use the sample data set for this tutorial and reconstruct source
+# activity on the trials with left auditory stimulation. Note that
+# beamformers are usually computed in a :class:`volume source space
+# <mne.VolSourceEsitmate>`, as a visualization
+# of only the surface activation can misrepresent the data.
```

Comment:
would be good to add a bit more detail here about the misrepresentation.  You say "beamformers are *usually* computed in a volume source space" (emphasis added), so are there legitimate reasons to sometimes use a cortical surface constrained estimate?  If so, under what conditions is that appropriate?

Reply:
In my view, it is not appropriate to use surface models. I have seen people do it, but do not know what their motivation is ...